src/components/SearchForm/SearchForm.js

```diff
@@ -11,11 +19,12 @@ function Search(props) {
           type="text"
           className="search__input"
           placeholder="Enter topic"
+          onChange={(e) => setKeyWord(e.target.value)}
```

Comment:
If you have `onChange` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about `controlled components`: https://reactjs.org/docs/forms.html#controlled-components